### PR TITLE
(#170) Organizations test

### PR DESCRIPTION
**Pull request by zniwiarzxxx**
tests for organizations service

### CODE REVIEW

**Comment by eggwhat** [comment]
@zniwiarzxxx mentioned changes are featured in #180

File: .github/workflows/cloud_deploy.yml [hunk 1]
```diff
@@ -3,10 +3,10 @@ name: Deploy to Cloud
 on:
   push:
     branches:
-      - dev
```
Comment (eggwhat) [inline .github/workflows/cloud_deploy.yml:6]: I am not sure from where do we want to deploy now. @SaintAngeLs

**Review by eggwhat — APPROVED** [review]
Tests work great. I had to add test projects to solution but it should be fixed now.

**Comment by SaintAngeLs** [comment]
I suspect, there is a need to update the scripts.

**Comment by SaintAngeLs** [comment]
> I suspect, there is a need to update the scripts.

No, there was no issue.

**Pull request merged**